### PR TITLE
Feature/rp comp

### PR DESCRIPTION
**Pull request by Tinkaa**
Place part of the return period functionalities in the `utils_general` folder. 
Such that it can be used by other datasets than GloFas as well.

### CODE REVIEW

File: src/utils_general/statistics.py
Comment (turnerm) [inline src/utils_general/statistics.py:8]: The method names should not begin with an underscore, as that is used to denote private methods (i.e. not used outside their own class / module)

File: src/indicators/flooding/glofas/utils.py
Comment (turnerm) [inline src/indicators/flooding/glofas/utils.py:111]: Sorry to be pedantic but there should be two blank lines between methods (I only noticed because PyCharm complains!)

**Review by turnerm — APPROVED** [review]
Looks great! But if you could please hold off on merging until we've merged the Nepal branch, otherwise it will become a total quagmire

**Comment by Tinkaa** [comment]
@turnerm will leave it to you to merge after you merged Nepal, alright?

**Comment by turnerm** [comment]
> @turnerm will leave it to you to merge after you merged Nepal, alright?

Will do!

**Pull request merged**
